docs/book/user-guide/tutorials/managing-scheduled-pipelines.md

```diff
@@ -0,0 +1,332 @@
+client.delete_schedule("daily-data-processing")
+
+# Create a new schedule with updated parameters
+new_schedule = Schedule(
+    name="daily-data-processing",
+    cron_expression="0 10 * * *"  # Changed to 10 AM
+)
+
+# Attach the new schedule to our pipeline
+updated_pipeline = daily_data_pipeline.with_options(schedule=new_schedule)
+
+# Run the pipeline to create the new schedule
+updated_pipeline()
+```
+
+Using the CLI to delete a schedule:
+
+```bash
+# Delete a specific schedule
+zenml pipeline schedule delete daily-data-processing
+```
+
+> **Important**: When updating schedules, you should also delete the
+> corresponding schedule in your orchestrator (Vertex AI in this example). You
+> can do this through the Google Cloud Console or using the orchestrator's API
+> (see below for code example).
+> ZenML's delete command may not always completely remove the underlying
+> orchestrator schedule.
+
+## Step 5: Monitor Schedule Execution
+
+Let's check the execution history of our scheduled pipeline:
+
+```python
+# Get recent pipeline runs
+runs = client.list_pipeline_runs(
+    pipeline_name_or_id="daily_data_pipeline",
+    sort_by="created",
+    descending=True,
+    size=5
+)
+
+print("Recent pipeline runs:")
+for run in runs.items:
+    print(f"Run ID: {run.id}")
+    print(f"Created at: {run.creation_time}")
+    print(f"Status: {run.status}")
+    print("---")
+```
+
+### Monitoring with Alerters
+
+For critical pipelines, [add alerting](https://docs.zenml.io/stacks/alerters) to notify you of failures:
+
+```python
+from zenml.hooks import alerter_failure_hook
+from zenml import pipeline, step
+
+# Add failure alerting to critical steps
+@step(on_failure=alerter_failure_hook)
+def critical_step():
+    # Step logic here
+    pass
+
+@pipeline()
+def monitored_pipeline():
+    critical_step()
+    # Other steps
+```
+
+This assumes you've registered an alerter (like Slack or Discord) in your active stack.
+
+## Step 6: Clean Up
+
+Finally, let's clean up by deleting our schedule:
+
+```python
+# Delete the schedule from ZenML
+client.delete_schedule("daily-data-processing")
+
+# Verify deletion
+schedules = client.list_schedules()
+if not any(s.name == "daily-data-processing" for s in schedules):
+    print("Schedule deleted successfully from ZenML!")
+else:
+    print("Schedule still exists in ZenML!")
+```
+
+> **Important**: You should also delete the schedule from the Vertex AI orchestrator to fully stop the scheduled runs. Here's how to delete Vertex AI schedules:
+
+```python
+from google.cloud import aiplatform
+
+# List all Vertex schedules
+vertex_schedules = aiplatform.PipelineJobSchedule.list(
+    filter='display_name="daily-data-processing"',
+    location="us-central1" # insert your location here
+)
+
+# Delete matching schedules
+for schedule in vertex_schedules:
+    print(f"Deleting Vertex schedule: {schedule.display_name}")
+    schedule.delete()
+```
+
+## Troubleshooting: Quick Fixes for Common Issues
```

Comment:
This is a great section. Maybe add some timezone stuff here because its a common misconception what timezones are used